lib/db.mysql.js

```diff
@@ -27,7 +27,7 @@ if (api.mysql) {
 
     connection.connect(err => {
       if (err) {
-        database.application.log.error(err);
+        database.application.log.error(err.toString());
```

Comment:
```suggestion
        database.application.log.error(err.stack);
```
?

Reply:
But need to check, will it optimize stack or not

Reply:
I mean optimize is shorten paths and remove line brakes and padding

Reply:
It will remove padding, but won't shorten paths: https://github.com/metarhia/metalog/blob/eb3f92c66f93ca5772563e047c7baded11770bb1/metalog.js#L58.